ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js

```diff
@@ -83,10 +83,8 @@ let getErrorResponse = function(event, jqxhr, settings, throwError) {
     settings.type === 'GET'
   ) {
     return {
-      title: `Error Polling ${properties.i18n['Sources']}`,
-      message: `Unable to query server for list of active ${
-        properties.i18n['sources']
-      }`,
+      title: properties.i18n['sources.polling.error.title'],
```

Comment:
✏️ I would suggest making the object keys more descriptive as to what type of `title` and `message` this is so it's easier to debug/refactor/find in the future

Reply:
I was hoping `source.polling.error` would be descriptive enough. It's just as much information as the error message used to give

Reply:
This seems descriptive enough to me. @leo-sakh did you have a concrete example of what you were looking for?